moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -207,14 +207,6 @@ void MotionPlanningDisplay::onInitialize()
 {
   PlanningSceneDisplay::onInitialize();
 
-  // Prepare database parameters
```

Comment:
Overwrites this commit: https://github.com/ros-planning/moveit2/commit/eb2c8b97862e9acb760dd0c7882d06d04e4ecfae